packages/bigquery/sodasql/dialects/bigquery_dialect.py

```diff
@@ -61,6 +62,33 @@ def create_connection(self):
         except Exception as e:
             self.try_to_raise_soda_sql_exception(e)
 
+    def __query_table(self, table_name):
+        query = f"""
+            SELECT *
+            FROM {table_name}
+            LIMIT 1
+        """
+        return query
+
+    def sql_test_connection(self, dataset_id) -> Union[Exception, bool]:
+        logging.info(f'Listing tables to check connection')
+        try:
+            tables = self.client.list_tables(dataset_id)
+            if tables:
+                logging.info("Tables contained in '{}':".format(dataset_id))
```

Comment:
We use versions of python that has format strings, you can change this to:

`f'Listing tables in {dataset_id}'`, same for other `"{}".format(...)` stuff below.